explanation/containers.md

```diff
@@ -1,12 +1,54 @@
-Containers are the centre piece of the Anbox Cloud stack. A single container provides a full Android system.
+Containers are the centre piece of the Anbox Cloud stack. Every time you launch an application or an image, Anbox Cloud creates a container for it. Every container provides a full Android system.
 
-Each container is hosted on a cluster of multiple nodes provided by the underlying LXD container hypervisor. The base for a container is an application. See [About applications](https://discourse.ubuntu.com/t/managing-applications/17760) for more information about applications.
+## Regular containers vs. base containers
+
+Anbox Cloud differentiates between two types of containers: regular containers and base containers. The container type is visible in the output of the `amc ls` command.
+
+Regular containers are containers that are launched from either an application or an image. They exist until they are deleted.
+
+Base containers are temporary containers that are used when [bootstrapping an application](https://discourse.ubuntu.com/t/managing-applications/17760#bootstrap). They are automatically deleted when the application bootstrap is completed.
+
+When we refer to containers in this documentation without specifying the container type, we mean regular containers.
+
+<a name="application-vs-raw"></a>
+## Application containers vs. raw containers
+
+Containers are based on either [applications](https://discourse.ubuntu.com/t/managing-applications/17760) or [images](https://discourse.ubuntu.com/t/provided-images/24185). That means that if you launch an application or an image, AMS automatically creates a container for it.
+
+Application containers, thus containers created when launching an application, run the full Android system. If the application is based on an Android app (an APK package), this app is launched after the system boots and monitored by the [watchdog](https://discourse.ubuntu.com/t/application-manifest/24197#watchdog). With the default configuration, you will only see the app and not the underlying Android system.
```

Comment:
`you will only see the app and not the underlying Android system.`. Do you mean the Android launcher ?

As I understand it right now with `Android system`, it means the user won't have access to any aspect of Android, which isn't true, as he can interact with the Android system via adb and some other means

Reply:
Not very familiar with Android terminology yet - but yes, I probably mean the launcher. ;) Namely that you can't just close the app and then see the "desktop".